extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/internal/discovery/NtpDiscovery.java

```diff
@@ -32,14 +32,14 @@
 /**
  *
  * The {@link NtpDiscovery} is used to add a ntp Thing for the local time in the discovery inbox
- * *
  *
  * @author Marcel Verpaalen - Initial contribution
+ * @author Alexander Kostadinov - Little binding improvements
  */
 @Component(service = DiscoveryService.class, immediate = true, configurationPid = "discovery.ntp")
 public class NtpDiscovery extends AbstractDiscoveryService {
 
-    public NtpDiscovery() throws IllegalArgumentException {
+    public NtpDiscovery() {

```

Comment:
The `super` constructor throws an IAE, so why not mark this one, that the IAE could be thrown?
Wouldn't it be helpful for the caller of the constructor to know that this runtime exception could be thrown?

Reply:
I removed it since I thought there is no sense explicitly marking it, but on a second thought you have a point - I will revert this change and push shortly.